src/index.js

```diff
@@ -1,26 +1,120 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
 import './styles.css';
+import { deleteTodoFromLS } from './modules/deleteTodoFromLS.js';
+import { addTodoToUI } from './modules/addTodoToUI.js';
+import { deleteTodoFromUI } from './modules/deleteTodoFromUI.js';
+
+class Store {
+  // getting todos from LS
+  static getTodos() {
+    let todos;
+    if (localStorage.getItem('Todos') !== null) {
+      todos = JSON.parse(localStorage.getItem('Todos'));
+    } else {
+      todos = [];
+    }
+    return todos;
+  }
+
+  // adding todo to local storage
```

Comment:
- [ ] Please remove the disabled linters and fix all the issues related to your code like you have too many classes, you should have just 1 as the linter set it.